serverless.yml

```diff
@@ -4,10 +4,11 @@ frameworkVersion: ">=1.1.0 <2.0.0"
 
 provider:
   name: aws
-  runtime: nodejs10.x
+  runtime: nodejs12.x
   memorySize: 128
   timeout: 60
-  stage: dev
+  region: ${env:AWS_REGION, env:AWS_DEFAULT_REGION, 'eu-west-2'}
```

Comment:
Here is how we deploy sls apps: https://github.com/comicrelief/drupal_concourse_tools/blob/master/scripts/sls_deploy/sls_deploy.sh
As you can see, the region is supposedly passed every time.
I don't think we use `eu-west-2`, it's either `eu-west-1` or `us-east-1`.
Personally I would say let it use default `eu-west-1` if we didn't state what to use, or just let it fail so that we can take notice.

Reply:
That's correct. The fallback is there just for deployments to the sandbox account (I used to deploy quite often the `serverless-user-service` to see whether the stack was correct).

I am happy to modify it to use `eu-west-1` tho, especially because some services are not available in `eu-west-2`.